### PR TITLE
Update utility package READMEs

### LINKED ISSUE #77: Improve utility package descriptions
**Is your feature request related to a problem? Please describe.**
In our GEL packages we could do with a better explanation of what GEL is, so anyone who sees the component knows what its trying to do.

**Describe the solution you'd like**
A clear and concise description of what GEL is including links on each package readme

- [ ] Initially labelled with ["Refinement needed"](https://github.com/BBC-News/psammead/labels/Refinement%20Needed)

### PR DESCRIPTION
**Pull request by bcmn**
Resolves #77 

Updates utility package READMEs, mainly to update descriptions, also to add some more detailed examples. **Still need to bump changelogs.**

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

**Comment by bcmn** [comment]
Blocked on #221, as it merges two of these packages.

**Comment by bcmn** [comment]
Unblocked. Will fix conflicts.

File: packages/utilities/gel-foundations/README.md [hunk 1]
```diff
@@ -15,21 +19,27 @@ npm install @bbc/gel-foundations --save
 ```jsx
 import { GEL_GROUP_3_SCREEN_WIDTH_MIN } from '@bbc/gel-foundations/breakpoints';
 
-import { GEL_GUTTER_BELOW_600PX } from '@bbc/gel-foundations/spacings';
+import { GEL_GUTTER_ABOVE_600PX } from '@bbc/gel-constants/spacings';
```
Comment (dr3) [inline packages/utilities/gel-foundations/README.md]: I don't think this is right, `gel-constants` is deprecated

File: packages/utilities/gel-foundations/README.md [hunk 2]
```diff
@@ -1,8 +1,12 @@
 # gel-foundations &middot; [![GitHub license](https://img.shields.io/badge/license-Apache%202.0-blue.svg)](https://github.com/BBC-News/psammead/blob/latest/LICENSE) [![npm version](https://img.shields.io/npm/v/@bbc/gel-foundations.svg)](https://www.npmjs.com/package/@bbc/gel-foundations) [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](https://github.com/BBC-News/psammead/blob/latest/CONTRIBUTING.md)
 
-This package is a collection of constants which can be imported into your application.
+This package provides a range of String constants for use in CSS, intended to help implement [BBC GEL-compliant](https://www.bbc.co.uk/gel/articles/what-is-gel) webpages and components.
```
Comment (dr3) [inline packages/utilities/gel-foundations/README.md]:
```suggestion
This package provides a range of string constants for use in CSS, intended to help implement [BBC GEL-compliant](https://www.bbc.co.uk/gel/articles/what-is-gel) webpages and components.
```

File: packages/utilities/gel-foundations/package.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "@bbc/gel-foundations",
-  "version": "0.1.2",
-  "description": "A collection of gel constants",
+  "version": "0.1.3",
+  "description": "A range of String constants for use in CSS, intended to help implement BBC GEL-compliant webpages and components.",
```
Comment (dr3) [inline packages/utilities/gel-foundations/package.json]:
```suggestion
  "description": "A range of string constants for use in CSS, intended to help implement BBC GEL-compliant webpages and components.",
```

File: packages/utilities/psammead-styles/README.md [hunk 4]
```diff
@@ -1,6 +1,11 @@
 # psammead-styles &middot; [![GitHub license](https://img.shields.io/badge/license-Apache%202.0-blue.svg)](https://github.com/BBC-News/psammead/blob/latest/LICENSE) [![npm version](https://img.shields.io/npm/v/@bbc/psammead-styles.svg)](https://www.npmjs.com/package/@bbc/psammead-styles) [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](https://github.com/BBC-News/psammead/blob/latest/CONTRIBUTING.md)
 
-This package is a collection of variable constants which can be imported into your application.
+This package provides a collection of String constants for use in CSS, containing non-GEL styling details that are bespoke to specific services.
+
+## Exports
+
+`/colours` - Project-defined colours that will be required by multiple Psammead components or themes. These colours are not defined by GEL.
+`/fonts` - Project-defined font families. These fallbacks are not defined by GEL.
```
Comment (dr3) [inline packages/utilities/psammead-styles/README.md:8]: Are these fallbacks?

File: packages/utilities/psammead-styles/package.json [hunk 5]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "@bbc/psammead-styles",
-  "version": "0.1.4",
-  "description": "A collection of BBC colours and fonts.",
+  "version": "0.1.5",
+  "description": "A collection of String constants for use in CSS, containing non-GEL styling details that are bespoke to specific services.",
```
Comment (dr3) [inline packages/utilities/psammead-styles/package.json]:
```suggestion
  "description": "A collection of string constants for use in CSS, containing non-GEL styling details that are bespoke to specific BBC services and products.",
```

**Review by dr3 — CHANGES_REQUESTED** [review]

File: packages/utilities/gel-foundations/README.md (hunk 1 above)
Comment (bcmn) [inline packages/utilities/gel-foundations/README.md]:
```suggestion
import { GEL_GUTTER_ABOVE_600PX } from '@bbc/gel-foundations/spacings';
```

Comment (bcmn) [reply packages/utilities/gel-foundations/README.md]: Addressed below.

File: packages/utilities/psammead-styles/README.md (hunk 4 above)
Comment (bcmn) [reply packages/utilities/psammead-styles/README.md:8]: The font fallbacks -- i.e. "Reith, then Helvetica". I was trying to communicate that these are specific to Articles, hence their inclusion in this package, & not `gel-foundations`. Any suggestions? 🤔

File: packages/utilities/psammead-styles/README.md [hunk 6]
```diff
@@ -1,6 +1,11 @@
 # psammead-styles &middot; [![GitHub license](https://img.shields.io/badge/license-Apache%202.0-blue.svg)](https://github.com/BBC-News/psammead/blob/latest/LICENSE) [![npm version](https://img.shields.io/npm/v/@bbc/psammead-styles.svg)](https://www.npmjs.com/package/@bbc/psammead-styles) [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](https://github.com/BBC-News/psammead/blob/latest/CONTRIBUTING.md)
 
-This package is a collection of variable constants which can be imported into your application.
+This package provides a collection of String constants for use in CSS, containing non-GEL styling details that are bespoke to specific services.
```
Comment (bcmn) [inline packages/utilities/psammead-styles/README.md]:
```suggestion
This package provides a collection of string constants for use in CSS, containing non-GEL styling details that are bespoke to specific BBC services and products
```

To. match Drew's suggestion.

**Review by dr3 — APPROVED** [review]

**Review by ChrisBAshton — APPROVED** [review]

**Pull request merged**